src/databricks/labs/lakebridge/assessments/dashboards/dashboard_manager.py

```diff
@@ -0,0 +1,85 @@
+import os
+import json
+
+import logging
+from pathlib import Path
+
+from databricks.sdk.service.dashboards import Dashboard
+from databricks.sdk.service.iam import User
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.blueprint.wheels import find_project_root
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Add a Comment to say why we are deviating from LSQL deployment, so other reviewers don't block this PR.

Reply:
Why are we deviating from LSQL for local dashboards?

LSQL has the following limitations:
- doesn't scale well to the many source EDWs Lakebridge will support
- limited support for visualization types beyond table, bar chart, and counter
- steep learning curve for future maintainers

Instead, we've adopted a dashboard template approach which can easily be mainained, easy to learn, serializable, and can be interpolated with the catalog and schema values.